test/mpi_runtests.jl

```diff
@@ -208,7 +208,12 @@ end
                     kwargs...
                 )
 
-                post_step = ProjectedEnergy(H, dv)
+                post_step = (
+                    ProjectedEnergy(H, dv),
+                    SignCoherence(copy(localpart(dv))),
+                    WalkerLoneliness(),
+                    Projector(proj_1=Norm2Projector()),
+                )
```

Comment:
Would it be useful to test passing an alternative name into `SignCoherence`?

Reply:
Good point, added.